src/null-safety/faq.md

```diff
@@ -175,17 +175,17 @@ An explicit runtime null check, for example `if (arg == null) throw
 ArgumentError(...)`, will be flagged as an unnecessary comparison if you make
 `arg` non-nullable.
 
-But, the check *is* still needed when running in mixed mode. Until everything is
-fully migrated and the code switches to running in sound mode, it will be
+But, the check *is* still needed when running with mixed versions. Until everything is
+fully migrated and the code switches to running with sound null safety, it will be
 possible for `arg` to be null.
 
 The simplest way to preserve behavior is change the check into
 [`ArgumentError.checkNotNull`](https://api.dart.dev/stable/dart-core/ArgumentError/checkNotNull.html).
 
-The same applies to some runtime type checks. In legacy or mixed mode, if arg
+The same applies to some runtime type checks. With mixed versions, if `arg`
```

Comment:
I'm not sure what the "With mixed versions," clause is trying to say here but I think it's clearest to just remove it.

Reply:
Thanks, @munificent!